src/setup/cortx_deploy

```diff
@@ -94,13 +101,49 @@ validate_config $CLUSTER_INFO
 
 export PATH=$PATH:/opt/seagate/provisioner/bin
 
-# Apply Cluster Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CLUSTER_INFO) -c "$CORTX_CONFSTORE";
+# Check system is pre-configured:
+# config apply: check confstore file.
+# cluster bootstrap: read overall_mp_status from state file.
 
-# Apply Common Config
-log $INFO "cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c '$CORTX_CONFSTORE'"
-cortx_setup config apply -f $(yaml_config $CONFIG_INFO) -c "$CORTX_CONFSTORE";
+
+FILE=$(awk -F:// '{print $2}' <<< $CORTX_CONFSTORE)
+if [ -f "$FILE" ]; then
+    log $INFO "File $FILE exists. System looks to be pre-configured."
+    CONFIG_APPLY=false
+fi
+local_path=`conf $CORTX_CONFSTORE get 'cortx>common>storage>local'`
+local_path=$(echo $local_path | tr -d "["\" | tr -d "\"]")
+node_id=`cat /etc/machine-id`
+STATE_FILE="$local_path/provisioner/state/status_$node_id"
+if [ -f "$STATE_FILE" ]; then
+    overall_mp_status=`conf json://$STATE_FILE get 'overall_mp_status'`
+    overall_mp_status=$(echo $overall_mp_status | tr -d "["\" | tr -d "\"]")
+    if [ "$overall_mp_status" = "success" ]; then
+        log $INFO "Components mini-provisioners stages are alredy executed."
+        CLUSTER_BOOTSTRAP=false
+    fi
+fi
+
+if [ -z $OVERRIDE ]; then
+    if [[ $CONFIG_APPLY = false && $CLUSTER_BOOTSTRAP = false ]]; then
+        log $INFO "System is already configured. exiting from cortx_deploy ..."
```

Comment:
This should be log WARN. Also if you can display some details like Cluster ID it would help user to correlate if he applied the command to a wrong cluster, etc.

Reply:
Done.
Added log statement (L-110) with cluster-id and node-id info.

`log $INFO "Starting deployment on node $node_id which is part of cluster $cluster_id."`